src/main/java/org/jabref/logic/citationstyle/CitationStyle.java

```diff
@@ -163,16 +163,22 @@ public String toString() {
     }
 
     @Override
-    public boolean equals(Object other) {
-        if (this == other) {
+    public boolean equals(Object o) {
```

Comment:
I renamed it to be consistent with the other implementations we have.
And there other is used as the name of the casted instance, see the line below